CHANGELOG.md

```diff
@@ -1,8 +1,20 @@
 # Changelog
 
-## [0.5.0](https://github.com/inviqa/kafka-consumer-go/tree/0.5.0) (2022-03-08)
+## [v0.6.0](https://github.com/inviqa/kafka-consumer-go/tree/v0.6.0) (2023-02-07)
 
-[Full Changelog](https://github.com/inviqa/kafka-consumer-go/compare/v0.4.0...0.5.0)
+[Full Changelog](https://github.com/inviqa/kafka-consumer-go/compare/v0.5.0...v0.6.0)
+
+**Merged pull requests:**
```

Comment:
is this generated? should it not be `improvements`  /  `features` etc?

Reply:
yes it's generated